### PR TITLE
Update mixer-tools structure and Makefile

### PR DESCRIPTION
**Pull request by tmarcu**
Follow the expected Go convention for packages, and put the project
under github.com/clearlinux/mixer-tools with the correct hierarchy.
Simplify Makfile for the time being.

Signed-off-by: Tudor Marcu <tudor.marcu@intel.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,31 +3,23 @@ VERSION = 3.2.0
...
+	install -m 00755 $(GOPATH)/bin/mixer $(DESTDIR)/usr/bin/.
 	install -m 00755 pack-maker.sh $(DESTDIR)/usr/bin/mixer-pack-maker.sh
 	install -m 00755 superpack-maker.sh $(DESTDIR)/usr/bin/mixer-superpack-maker.sh
 	install -D -m 00644 yum.conf.in $(DESTDIR)/usr/share/defaults/mixer/yum.conf.in
 
 release:
 	git archive --format=tar.gz --verbose -o mixer-tools-$(VERSION).tar.gz HEAD --prefix=mixer-tools-$(VERSION)/
 
-all: compliant $(BUILDABLES)
+all: compliant
```
Comment (icarus-sparry) [inline Makefile]: Are we expecting developers to be running make, or just devops? If developers are then a few more targets might be helpful, e.g. one to run metalinter and one to build without go vet.

File: Makefile [hunk 2]
```diff
@@ -3,31 +3,23 @@ VERSION = 3.2.0
...
+	go install ./...
+
+clean:
+	rm -rf $(GOPATH)/bin/mixer
+	rm -rf mixer-tools-*.tar.gz
```
Comment (icarus-sparry) [inline Makefile]: Maybe use ```go clean``` with its various options?

File: Makefile (hunk 1 above)
Comment (tmarcu) [reply Makefile]: I intend to add that but not as part of this commit. This is more for devops, not so much developers IMHO.

File: Makefile [hunk 3]
```diff
@@ -3,31 +3,23 @@ VERSION = 3.2.0
...
-# We want to add compliance for all built binaries
-_CHECK_COMPLIANCE = $(addsuffix .compliant,$(_COMPLIABLE))
-
 # Ensure our own code is compliant..
-compliant: $(_CHECK_COMPLIANCE)
-install: $(BINS)
-	test -d $(DESTDIR)/usr/bin || install -D -d -m 00755 $(DESTDIR)/usr/bin; \
-	install -m 00755 bin/* $(DESTDIR)/usr/bin/.
+compliant:
+	go fmt ./... && go vet ./...
```
[the remark this answers is not in the document]
Comment (tmarcu) [reply Makefile]: yes this should clearly be checked :) This section will likely expand with the heavier linter options in the near future as well.

**Pull request merged**